staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4911,6 +4911,15 @@ const (
 	ServiceExternalTrafficPolicyTypeCluster = ServiceExternalTrafficPolicyCluster
 )
 
+// These are valid values for the TrafficDistribution field of a Service.
+const (
+	// Indicates a preference for routing traffic to endpoints that are
+	// topologically proximate to the client. The interpretation of "topologically
+	// proximate" may vary across implementations and could encompass endpoints
+	// within the same node, rack, zone, or even region.
+	ServiceTrafficDistributionPreferClose = "PreferClose"
```

Comment:
the kep says `Close`, I'm late here, have I missed some follow ups discussions?

Reply:
I'm actually pretty happy with the name and value proposed in this KEP, thanks for updating it here!

Reply:
Rob - do you mean in this PR or in the KEP?

Reply:
🤦 meant "PR" - I like the terminology proposed by Gaurav here. Sorry for the confusion - updated my initial comment for the sake of future readers.